door_adapter_template/config.yaml

```diff
@@ -2,18 +2,26 @@
 # RMF door parameters
 
 ### template ###
-door:
-  name: "example_door"
-  api_endpoint: "http://127.0.0.1:8888/"
-  header_key: "example_key"
-  header_value: "example_header_value"
-  door_id: "example_door_id"
-  door_signal_period: 3.0 # Seconds
-  door_close_feature: False 
+doors:
+  "door_1":  # door name and id
+    door_close_feature: False
```

Comment:
Looking at the implementation, `door_close_feature` refers to the adapter requesting the door to close explicitly, rather than just stop publishing door open requests. I initially thought it meant that the door will automatically close.

`close_door_explicitly` will be clear, but it doesn't describe how a door will close automatically when door open requests are stopped

what do you think of `auto_close` or `door_auto_closes`? It describes that the door will close on its own automatically after a while, which is why constant open requests are required, and also explains why no explicit door close requests are required.

Reply:
Updated the parameter names like you suggested, also put in some comments explaining what they mean. After discussion we are going to use `door_auto_closes` to represent a door that automatically closes, but continue to support older configs that still use `door_close_feature`. It should be noted that we're deprecating the latter, and that opposite booleans should be used to represent them, i.e. if `door_close_feature = True`, it means that the door requires explicit door close requests, so `door_auto_closes = False`.